web_src/js/features/repo-issue-list.js

```diff
@@ -69,16 +69,14 @@ function initRepoIssueListCheckboxes() {
       }
     }
 
-    updateIssuesMeta(
-      url,
-      action,
-      issueIDs,
-      elementId,
-    ).then(() => {
+    try {
+      await updateIssuesMeta(url, action, issueIDs, elementId);
       window.location.reload();
-    }).catch((reason) => {
-      showErrorToast(reason.responseJSON.error);
-    });
+    } catch (err) {
+      if (err?.responseJSON?.error) {
+        showErrorToast(err.responseJSON.error);
+      }
```

Comment:
Add a "else" ? Ignoring errors silently might hide some bugs.

Reply:
Good point. Even thought `window.location.reload` is unlikely to throw, we can add it.